src/exprs.h

```diff
@@ -25,10 +27,15 @@
 **  'OBJ_REFLVAR'  returns  the value of  the reference  to a  local variable
 **  <expr>.
 */
-#define OBJ_REFLVAR(expr)       \
-                        (OBJ_LVAR( LVAR_REFLVAR( expr ) ) != 0 ? \
-                         OBJ_LVAR( LVAR_REFLVAR( expr ) ) : \
-                         ObjLVar( LVAR_REFLVAR( expr ) ) )
+static inline Obj OBJ_REFLVAR(Expr expr)
+{
+    if (OBJ_LVAR(LVAR_REFLVAR(expr)) != 0) {
+        return OBJ_LVAR(LVAR_REFLVAR(expr));
+    }
+    else {
+        return ObjLVar(LVAR_REFLVAR(expr));
+    }
```

Comment:
So I understand that this is the most straightforward implementation, but since we are no in a function, we could also untangle the code to make it more palatable, e.g. like this:
```C
    Int lvar = LVAR_REFLVAR(expr);
    Obj obj = OBJ_LVAR(lvar);
    return obj ? obj : ObjLVar(lvar);
```

Reply:
fixed (well, partially, I'd prefer keeping the if)